frontend/src/components/UserUpdateRoles.vue

```diff
@@ -81,24 +81,31 @@
           </v-row>
         </v-col>
       </v-row>
-      <div class="my-6" v-if="selectedClientId">
-        <v-btn id="save-user-roles" class="secondary" medium v-on:click="updateUserClientRoles()">Save User Roles</v-btn>
+      <div class="my-6" v-if="selectedClient">
+        <v-btn id="save-user-roles" class="primary" medium v-on:click="updateUserClientRoles()">Save User Roles</v-btn>
       </div>
+      <span v-if="selectedClient.clientId.includes('SFDS')">
+        <v-divider class="sub-permissions"></v-divider>
+        <user-update-sfds-auth></user-update-sfds-auth>
+      </span>
     </div>
+
   </v-card>
 </template>
 
 <script>
 import ClientsRepository from "@/api/ClientsRepository";
 import UsersRepository from "@/api/UsersRepository";
+import UserUpdateSfdsAuth from "@/components/UserUpdateSfdsAuth";
 
 export default {
   name: "UserUpdateRoles",
+  components: { UserUpdateSfdsAuth },
   props: ['userId'],
   data() {
     return {
       clients: [],
-      selectedClientId: null,
+      selectedClient: null,
```

Comment:
map the actual client object rather than just the ID from the autocomplete menu
this lets us check the name to see if it matches the SFDS client to display the additional component